pkg/extract/extract.go

```diff
@@ -13,6 +14,7 @@ import (
 	"github.com/cheggaaa/pb/v3"
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/h2non/filetype"
+	"github.com/klauspost/compress/zstd"
```

Comment:
Didn't go through the PR but as per document `Currently the package is heavily optimized for 64 bit processors and will be significantly slower on 32 bit processors.`  and some folks want to use crc on power-pc does it also affect performance there or they should be good using tarball of xz compression?

Reply:
I expect most power machines to be 64 bit? Name of the arch is `ppc64le` after all. Not sure about s390, but I'd expect them to also be 64 bit.

Reply:
we do not support 32bits anyways, as Hyper-V is not available on those platforms (our image is 64bits), plus even PPC has been 64bits long before Intel+AMD started testing this.